### PR TITLE
Extracting the data values processing & displaying totals

### PR DESCRIPTION
**Pull request by sejpat25**
The components used to display this information had to change according to our dashboard display. Now data associated with the nodes are received and processed to display the maximum, accumulated sum and average of a particular value associated to a node.

### CODE REVIEW

File: src/components/DataTable/DropDownNodeMenu/DropDownNodeMenu.js [hunk 1]
```diff
@@ -3,30 +3,12 @@ import DropDownMenu from 'material-ui/DropDownMenu';
 import MenuItem from 'material-ui/MenuItem';
```
Comment (Edds390) [inline src/components/DataTable/DropDownNodeMenu/DropDownNodeMenu.js]: Remove unused imports

File: src/components/DataTable/DropDownNodeMenu/DropDownNodeMenu.js [hunk 2]
```diff
@@ -3,30 +3,12 @@ import DropDownMenu from 'material-ui/DropDownMenu';
...
 import ExpandableCard from '../ExpandableCard/ExpandableCard';
 import Paper from 'material-ui/Paper';
```
Comment (Edds390) [inline src/components/DataTable/DropDownNodeMenu/DropDownNodeMenu.js]: absolute imports before relative imports

**Review by Edds390 — CHANGES_REQUESTED** [review]
Very good commenting and easy to follow code structure. Minor changes to import statements requested. Also, the paragraph bodies of the statistics are using the wrong font:
image
Override using css requested.

**Review by Edds390 — APPROVED** [review]

**Comment by Edds390** [comment]
Closes [Issue 21](https://github.com/Edds390/solarnetwork-aggregated-dashboard/issues/21)

**Review by KenneyC — APPROVED** [review]
Tested on your branch, looks good to me 👍 Fantastic work!

**Pull request merged**